src/main/python/design_view.py

```diff
@@ -0,0 +1,330 @@
+        self.model = model
+
+        layout = QtWidgets.QGridLayout()
+
+        self.default_width = 260
+        self.default_height = 320
+        self.pos = QtCore.QPoint(model.x, model.y)
+        self.offset = QtCore.QPoint(0, 0)
+        self.title = QtWidgets.QLabel(self.name)
+        self.title.setMinimumHeight(24)
+        self.title.setMaximumHeight(24)
+
+        layout.addWidget(self.title, 0, 0)
+        layout.addWidget(self.view, 1, 0)
+        layout.addWidget(QtWidgets.QLabel("Next"), 2, 0, 1, 1, QtCore.Qt.AlignRight)
+
+        self.setLayout(layout)
+        self.setGeometry(model.x, model.y, model.width, model.height)
+
+    def contextMenuEvent(self, event):
+        contextMenu = QtWidgets.QMenu(self)
+        # edit = contextMenu.addAction("Edit")
+        # change_type = QMenu("Change type")
+        # contextMenu.addMenu(change_type)
+        # change_to_email = change_type.addAction("Send Email")
+        # change_to_reminder = change_type.addAction("Reminder")
+        # change_to_schedule = change_type.addAction("Schedule")
+        delete = contextMenu.addAction("Delete")
+        action = contextMenu.exec_(self.mapToGlobal(event.pos()))
+        # if action == edit:
+        # self.process_editor.process_text_edit.setText(self.query)
+        if action == delete:
+            self.process_editor.remove_process(self, self.model)
+        self.process_editor.update()
+
+    def mousePressEvent(self, event):
+        if event.buttons() & QtCore.Qt.LeftButton:
+            self.raise_()
+            self.offset = event.pos()
+        elif event.buttons() & QtCore.Qt.RightButton:
+            self.contextMenuEvent(event)
+
+    def mouseMoveEvent(self, event):
+        if event.buttons() & QtCore.Qt.LeftButton:
+            self.pos = self.mapToParent(event.pos() - self.offset)
+            self.move(self.pos)
+            self.parent().update()
+
+    def setup(self, layout):
+        return
+
+
+class SendEmailView(QtWidgets.QFrame):
+    def __init__(self, model):
+        super(SendEmailView, self).__init__()
+        layout = QtWidgets.QVBoxLayout()
+        layout.setContentsMargins(0, 0, 0, 0)
+        self.model = model
+
+        self.recipients = QtWidgets.QLineEdit(", ".join(model.recipients))
+        self.recipients.setPlaceholderText("Recipient1, Recipient2, ...")
+        self.recipients.textChanged.connect(model.setRecipients)
+
+        self.when = QtWidgets.QDateTimeEdit(QtCore.QDateTime(model.when))
+        self.when.dateTimeChanged.connect(self.setWhen)
+
+        self.body = QtWidgets.QTextEdit(model.body)
+        self.body.setPlaceholderText("Enter the body...")
+        self.body.textChanged.connect(model.setBody)
+
+        layout.addWidget(self.recipients)
+        layout.addWidget(self.when)
+        layout.addWidget(self.body)
+
+        self.setLayout(layout)
+
+    def setWhen(self, dt):
+        self.model.setWhen(dt.toPyDateTime())
+
+
+# NOTE(alexander): DEV mode entry point only!!!
```

Comment:
Probably shouldn't push comments to master.

Reply:
I think that we should keep this comment, so that everyone knows why this entry point is here and so that we don't forget what it is used for.

Reply:
We probably want to remove this entry point before release but for the time being it is nice to have when developing the GUI.